src/blog/singlePage.html

```diff
@@ -77,19 +77,22 @@
                     <a href="/">Home</a>
                   </li>
                   <li>
-                    <a href="index.html#services">Services</a>
+                    <a href="#services">Services</a>
```

Comment:
agree with creating single reusable navbar and will create an asana task for this